### DISCUSSION TITLE
Images not loading on deployment

### DISCUSSION THREAD

**Discussion by VishvamPorwal** [vikasdo/Book-Recommendation-Analysis#99] [Ideas]
I found out two solutions
1. This may not be the case
![Screenshot_20210421_135506](https://user-images.githubusercontent.com/59139142/115521865-33f81080-a2a9-11eb-8693-3a268f830f5c.png)

2. maybe this setup is needed [link of heroku article](https://devcenter.heroku.com/articles/s3-upload-python)
![Screenshot_20210421_135620](https://user-images.githubusercontent.com/59139142/115522020-5be77400-a2a9-11eb-92af-4ae3bd260191.png)
the second might be the case
so sir, I was wondering if you know something about this.

**Comment by vikasdo** [comment]
We are not using any Aws images in our website

**Comment by VishvamPorwal** [comment]
![Screenshot_20210427_142139](https://user-images.githubusercontent.com/59139142/116213995-e6c1e600-a763-11eb-88cd-9124864da427.png)
aren't these aws images

**Comment by vikasdo** [comment]
Yes sorry these are Aws images only

**Comment by vikasdo** [comment]
But these are public open to all we need not to configure it with
credentials Access key

**Reply by VishvamPorwal** [reply]
okay i will keep looking then, there might be some other problem
